packages/module-services/src/module.ts

```diff
@@ -0,0 +1,58 @@
+import type { Module } from '@equinor/fusion-framework-module';
+import type { HttpModule } from '@equinor/fusion-framework-module-http';
+import type {
+    ServiceDiscoveryModule,
+    ServiceDiscoveryProvider,
+} from '@equinor/fusion-framework-module-service-discovery';
+
+import { IApiConfigurator, ApiConfigurator } from './configurator';
+import { IApiProvider, ApiProvider } from './provider';
+
+export type ServicesModuleKey = 'services';
+
+export type ServicesModule = Module<
+    ServicesModuleKey,
+    IApiProvider,
+    IApiConfigurator,
+    [HttpModule, ServiceDiscoveryModule]
+>;
+
+/**
+ *  Configure http-client
+ */
+export const module: ServicesModule = {
+    name: 'services',
```

Comment:
du har `ServicesModuleKey` så hvorfor ikke bruke den ;)

Reply:
`type` is only `type`:)